src/runtime_src/core/pcie/tools/xbmgmt/xbmgmt.cpp

```diff
@@ -48,6 +48,7 @@ static const std::map<std::string, struct subCmd> subCmdList = {
     { "partition", {partHandler, subCmdPartDesc, subCmdPartUsage} },
     { "config", {configHandler, subCmdConfigDesc, subCmdConfigUsage} },
     { "nifd", {nifdHandler, subCmdNifdDesc, subCmdNifdUsage} },
+    { "-hotplug", {hotplugHandler, subCmdHotplugDesc, subCmdHotplugUsage} },
```

Comment:
no '-' please.

Reply:
'-' option to make this command hidden. This is the implementation follows to hide this command from the help menu.  

One more command has there for same. 
{ "-lash", {flashXbutilFlashHandler, subCmdXbutilFlashDesc, subCmdXbutilFlashUsage} },

Reply:
This command should not be hidden. It is an expert only command. The xbmgmt help should show it as expert only command. Please post the output of xbmgmt help, and xbmgmt help --expert with your change.

Reply:
This is suggested by Sonal that this option should make as hidden.

Reply:
"-lash" is a very special case on its own. it's not meant to be used by anyone or anywhere other than in xbutil.cpp for backward compatibility purpose and will be removed in the future.
the rest of options are command line options which is divided into normal and expert-only groups. all expert-only subcommands are not documented and should only show up in help message when the undocumented "--expert" option is used. thus, all expert-only subcommands are hidden.
the "--expert" option is newly added. please sync up with master branch and retest your change.